internal/api/plugin.go

```diff
@@ -37,13 +107,23 @@ func (s *Server) GetPlugins(c echo.Context) error {
 func (s *Server) GetPlugin(c echo.Context) error {
 	pluginID := c.Param("pluginId")
 	if pluginID == "" {
-		return c.JSON(http.StatusBadRequest, NewErrorResponse("failed to get plugin"))
+		err := fmt.Errorf("plugin id is required")
+		message := echo.Map{
```

Comment:
Also this , we changed it to follow the pattern on the left side   , can you please make sure you don't revert it

Reply:
yes have fixed it in all the places